src/commands/dev/index.js

```diff
@@ -424,6 +424,7 @@ class DevCommand extends Command {
     const envFile = path.resolve(site.root, '.env')
     if (fs.existsSync(envFile)) {
       const vars = dotenv.parse(fs.readFileSync(envFile)) || {}
+      settings.env = { ...settings.env, ...vars }
       console.log(`${NETLIFYDEVLOG} Overriding the following env variables with ${chalk.blue('.env')} file:`, chalk.yellow(Object.keys(vars)))
       Object.entries(vars).forEach(([key, val]) => process.env[key] = val)
```

Comment:
`process.env[key]` being set here. Is not accessible two lines down when accessed in [`startDevServer`](https://github.com/netlify/cli/pull/764/files#diff-8a2f65e3aede3ea9696e546766f50f03R319).
cc @erquhart

Reply:
The cause for this bug is the spread order of properties here:
https://github.com/netlify/cli/blob/3085892a7fb54f46c954ab82839dce605cf44cba/src/commands/dev/index.js#L341

`settings.env` properties override `process.env` properties.

Changing that line to `env: {  ...settings.env,...process.env, FORCE_COLOR: 'true' },`  fixes it